packages/client/src/app/pages/search/search.component.html

```diff
@@ -1,5 +1,142 @@
-<div class="empty">
-    <h3>Pardon the dust!</h3>
-    <p>This feature has been temporarily disabled.</p>
-    <p>Keep an eye on the patch notes to see when this is coming back!</p>
-</div>
\ No newline at end of file
+<div class="search-nav">
+    <div class="search-bar">
+        <form [formGroup]="searchForm" (ngSubmit)="submitSearch()">
+            <input formControlName="query" type="text" class="u-full-width" placeholder="Search ...">
+        </form>
+    </div>
+</div>
+
+<ng-container *ngIf="route.children.length === 0">
```

Comment:
Note about this page: all three search result types are very visually different. I did a little bit of cleanup here, but not much.